### ISSUE TITLE
Regression: Formatting multiline calls with named arguments

### ISSUE THREAD

**Issue by krlmlr** [r-lib/styler#369]
I can work around this for now, but we should be adding indention here too.

``` r
styler::style_text("call(named = c, \nnamed = b)", strict = FALSE)
#> call(named = c,
#> named = b)
```

> Created on 2018-03-13 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).

**Comment by lorenzwalthert** [comment]
Ok, this seems to be a bug introduced with styler 1.0.1

**Comment by lorenzwalthert** [comment]
It seems to work fine with unnamed arguments though:
``` r
styler::style_text("call(1, \n2)", strict = FALSE)
#> call(1,
#>   2)
```

Created on 2018-03-13 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).

**Comment by lorenzwalthert** [comment]
Ok, I think I have identified the problem and just opened #372. The problem is just that it is too complicated, there are many edge cases and I keep forgetting how it works 😩.  So not 100% sure if my solution does not introduce another problem. I think if we fix problems like this, we should also extend the documentation and add examples that show why rules are implemented in a certain way (as I did in #372).
